Update/some_004.txt

```diff
@@ -3789,7 +3789,7 @@ void main()
 //r吐き捨てるように呟き、千紗登ちゃんは奥歯をかみ締めるように憤怒の表情を浮かべる。そしてきっ、と冷たい視線を私に向けた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "吐き捨てるように呟き、千紗登ちゃんは奥歯をかみ締めるように憤怒の表情を浮かべる。そしてきっ、と冷たい視線を私に向けた。",
-			NULL, "Muttering that like she was about to spit on me, Chisato-chan bit hard on her back teeth and looked at me with rage in her eyes.", Line_Normal);
+			NULL, "As she spat those words muttering, Chisato-chan bit back hard on her teeth and gazed at me coldly.", Line_Normal);
```

Comment:
```suggestion
			NULL, "As she spat out those words, Chisato-chan bit back hard on her teeth and gazed at me coldly.", Line_Normal);
```
"Spat" and "mutter" aren't really compatible verbs. We should go with one or the other. Because sometimes the soul of translation is omission. :)

Reply:
```suggestion
			NULL, "As she spat those words out, Chisato-chan tightly clenched her teeth and coldly gazed at me.", Line_Normal);
```